symphony-bdk-http/symphony-bdk-http-webclient/src/main/java/com/symphony/bdk/http/webclient/ApiClientWebClient.java

```diff
@@ -146,7 +147,9 @@ public ApiClientWebClient(final WebClient webClient, String basePath, Map<String
       if (unwrap instanceof ApiException) {
         throw (ApiException) unwrap;
       } if(e.getCause() instanceof ConnectTimeoutException){
-        throw new RuntimeException(new SocketTimeoutException(e.getMessage()));
+        WebClientRequestException exception = (WebClientRequestException) e;
```

Comment:
ConnectTimeoutException should be always encapsulated by a WebClientRequestException in the webclient but just to avoid issues I will add a check in the if statement.